src/main/java/org/dcsa/bkg/repository/BKGConfirmationSummaryTORepository.java

```diff
@@ -0,0 +1,12 @@
+package org.dcsa.bkg.repository;
+
+import org.dcsa.bkg.model.transferobjects.BookingConfirmationSummaryTO;
+import org.dcsa.core.repository.ExtendedRepository;
+import org.springframework.data.r2dbc.repository.Query;
+import reactor.core.publisher.Flux;
+
+import java.util.UUID;
+
+public interface BKGConfirmationSummaryTORepository extends ExtendedRepository<BookingConfirmationSummaryTO, UUID> {

```

Comment:
Not required if we set up shipment repository.

Reply:
Should this repository be created in the event-core?